.devcontainer/customisations-team/devcontainer.json

```diff
@@ -1,12 +1,12 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.238.1/containers/ubuntu
 {
-	"name": "Ubuntu",
+	"name": "phpBB (Customisations Team)",
 	"build": {
-		"dockerfile": "Dockerfile",
-		// Update 'VARIANT' to pick an Ubuntu version: jammy / ubuntu-22.04, focal / ubuntu-20.04, bionic /ubuntu-18.04
-		// Use ubuntu-22.04 or ubuntu-18.04 on local arm64/Apple Silicon.
-		"args": { "VARIANT": "ubuntu-22.04" }
+		"dockerfile": "../Dockerfile",
+		"args": { 
+			"VARIANT": "ubuntu-22.04"
```

Comment:
any reason for 22.04 instead of 24.04?

Reply:
This is expected. There's a [known issue with 24](https://github.com/devcontainers/images/issues/1056) which is why I left it on 22.